input/i18n/de/my-account.yaml

```diff
@@ -21,9 +21,19 @@ personalDetails:
   welcomeBack: Willkommen zurück
   welcomeDescription: Für einen besseren Kundenservice geben Sie bitte Ihre Kundennummer an.
   personalDetailsTitle: Ihre Benutzerdaten
-shippingAddress:
-  sameBillingAddressCheckboxLabel: Als meine Rechnungadresse verwenden
-  sameDeliveryAddressCheckboxLabel: Als meine Lieferadresse verwenden
+addressBook:
+  useAsDefaultBillingAddressCheckboxLabel: Als meine Rechnungadresse verwenden
+  useAsDefaultShippingAddressCheckboxLabel: Als meine Lieferadresse verwenden
+  contentTitle : Addressbuch
```

Comment:
As said in the other PR, better call this `title` and make it first of the list.